src/NUnitEngine/nunit.engine/Services/ExtensionService.cs

```diff
@@ -431,89 +431,99 @@ internal void FindExtensionsInAssembly(ExtensionAssembly assembly)
         {
             log.Info("Scanning {0} assembly for Extensions", assembly.FilePath);
 
-            ValidateTargetFramework(Assembly.GetEntryAssembly(), assembly);
+            if (ValidateTargetFramework(Assembly.GetEntryAssembly(), assembly))
```

Comment:
Can we rename this method, now we're changing the use of it a little? Maybe something like `CanLoadTargetFramework`?